tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -132,19 +157,22 @@ func TestRandomizedSmall(t *testing.T) {
 
 	st := timetrack.StartTimer()
 
+	maxDirDepth := 3
+
 	opts := engine.ActionOpts{
 		engine.ActionControlActionKey: map[string]string{
 			string(engine.SnapshotDirActionKey):              strconv.Itoa(2),
 			string(engine.RestoreSnapshotActionKey):          strconv.Itoa(2),
-			string(engine.DeleteRandomSnapshotActionKey):     strconv.Itoa(1),
+			string(engine.DeleteRandomSnapshotActionKey):     strconv.Itoa(4),
```

Comment:
Why change this from 1 to 4?

Reply:
This will increase the weight for deleting random snapshots from the repository under test. Increasing the weight will result in deletion of more snapshots. This will maintain a smaller delta in the number of snapshots added in each test run. Thus reducing the filesystem capacity required to run the test long term.

Reply:
What happens when the weight for deleting snapshots is (much) larger than snapshot creation?

Reply:
From the point of view of this specific test, it would run out of snapshots at some point. If we consider rest of tests in the suite, that may not happen. I have reverted the weight increase because it is not the best idea to rely on other test methods to create more snapshots than those are deleted here.

Reply:
True, other parts of the job may create additional snapshots.
Nevertheless, we should be aware of the implications of the change. It may be a good idea to do minor tweaks to the weights and monitor the number of snapshots over time.
We can discuss this in a separate PR.
Thanks.